source/scenario/blockly/mission3c.xml

```diff
@@ -32,6 +32,6 @@
   </category>
   <sep></sep>
   <category name="Procedures">
-    <block type="variables_get" inline="false"><field name="VAR">y-coordinate</field></block>
+    <block type="variables_get" inline="false" editable="false"><field name="VAR">y-coordinate</field></block>
```

Comment:
Could we change the name of these while we're messing with them?  Destination X and Destination Y...